flask_github.py

```diff
@@ -149,8 +233,12 @@ def raw_request(self, method, resource, params=None, **kwargs):
         if params is None:
             params = {}
 
-        if 'access_token' not in params:
-            params['access_token'] = self.get_access_token()
+        # Set ``Authorization`` header, ``access_token`` query parameter
+        if self.get_access_token():
+            params.setdefault('access_token', self.get_access_token())
+            kwargs.setdefault('headers', {})
+            kwargs['headers'].setdefault('Authorization',
+                                         'token %s' % self.get_access_token())
```

Comment:
```self.get_access_token()``` is called more than once. User may be reading the token from database or some other slow source. Can we just call it once and keep the token in a variable?